### DISCUSSION TITLE
IntelliJ and Maven examples?

### DISCUSSION THREAD

**Discussion by VGPS** [deepjavalibrary/djl#3124] [General]
Hi,
I use IntelliJ and maven primarily for Java development. I see this is geared to Eclipse and gradle. Is there a sample maven.xml file and instructions for running the code in Eclipse? Thanks!

**Comment by frankfliu** [comment]
You can use IntelliJ with gradle. Most of our developers are using IntelliJ, see: https://github.com/deepjavalibrary/djl/blob/master/docs/development/development_guideline.md#run-examples-in-intellij
